pinot-segment-local/src/main/java/org/apache/pinot/segment/local/data/manager/SegmentDataManager.java

```diff
@@ -27,6 +27,15 @@
  */
 public abstract class SegmentDataManager {
   private int _referenceCount = 1;
+  private long _segmentLoadTimeInMillisEpoch = System.currentTimeMillis();
```

Comment:
(minor) `_loadTimeMs` for simplicity, same for the getter and setter

Reply:
Ack.